todolist/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+
+from .models import *
```

Comment:
* syntax is considered as bad practice. I offer to import required objects only

Reply:
Sure, it's not finished yet